actions/portable_testing/Makefile

```diff
@@ -0,0 +1,71 @@
+# Copyright 2025 The Sigstore Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# optionally set these env variables
+FULCIO_REPO ?= https://github.com/sigstore/fulcio.git
+REKOR_REPO ?= https://github.com/sigstore/rekor.git
```

Comment:
The main reason we want this action is to test rekor-tiles, can we go ahead and use that here? Or make it configurable?

Reply:
Eventually yes, but iiuc rekor-tiles has a slightly different APIs that clients like cosign are not yet ready to use.

Reply:
But they don't have to use this, they can use what they already have. They need this in order to update.

Reply:
Ok I see what you mean: Have this Makefile is rekor-tiles as default, and clients may optionally use rekor-v1 by setting the REKOR_REPO env variable. Right?

Reply:
Yes. But I think many clients may not bother to use this for rekor v1 because they already have their own ways of testing against rekor v1.